src/app/pages/Avisos/index.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react';
+import { TotalsContainer } from './elements';
+import Radio from 'app/components/ui/Radio';
+import AvisosElement from 'app/components/ui/AvisosElement';
+import FilterContainer from 'app/components/ui/PaddedContainer';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Typography from '@material-ui/core/Typography';
+import { avisos } from 'app/responses';
+
+class Avisos extends Component{
+  constructor(){
```

Comment:
No es necesario crear un `constructor`
```suggestion
  class Avisos extends Component {
     state = {
       filterBy: 'date',
       ...
     }
```

Reply:
El problema es con la llamada a la función groupBy2() que necesita que el `this` esté andando. Hay otra opción?

Reply:
no entiendo bien a que the refires, haciendo
```suggestion
class Some extends Component {
   state = { x: y };
}
```
state es parte de la clase Some. Puedes hacer `this.state` dentro de la clase. En caso de que te refieras a eso.

Reply:
No, mira en la línea 15 que estoy llamando a un método del componente.